src/control/Control.Layers.js

```diff
@@ -248,6 +248,16 @@ L.Control.Layers = L.Control.extend({
 
 	_expand: function () {
 		L.DomUtil.addClass(this._container, 'leaflet-control-layers-expanded');
+		var getLegendDiv = document.getElementsByClassName('leaflet-control-layers-list')[0];
+		var legendHeight = getLegendDiv.clientHeight;
+		var getMap = document.getElementById('map');

```

Comment:
Map height can be accessed with `this._map.getSize().y`.
Also `getSomething` names are misleading — usually you would call a function like this, not a map.